x-pack/plugins/integrations_manager/README.md

```diff
@@ -0,0 +1,38 @@
+# Integrations Manager
+
+## Directory overview
+```
+x-pack/plugins/integrations_manager/
+├── common 
+│   ├── constants.ts        # common strings, mostly. id, paths, etc
+│   └── types.ts            # as we find common types
+├── index.ts                # legacy plugin definition
+├── kibana.json             # new platform style shown in MIGRATION.md
+├── public
+│   ├── components          # plugin-specific components (used by screens/*)
+│   ├── data.ts             # all client-side data fetching
+│   ├── index.tsx           # React root. Client-side routes applied
+│   ├── register_feature.ts # add to registry for /#/home/feature_directory
+│   ├── routes.tsx          # every re: client-side routes. paths, mappings
+│   └── screens             # typically a view for a client-side route
+│       ├── detail.tsx      # integration detail view i.e. '/detail/:pkgkey'
+│       └── home.tsx        # integration list view i.e. '/'
+└── server            
+    ├── index.ts            # new platform style shown in MIGRATION.md 
+    ├── plugin.ts           # new platform style shown in MIGRATION.md
+    ├── registry.ts         # anything registry-related
+    ├── routes.ts           # every re: server-side routes. paths, mappings
+    └── saved_objects.ts    # anything saved objects-related
+```
+
+## Development
+We're using a long-running feature branch [`feature-integrations-manager`](https://github.com/elastic/kibana/tree/feature-integrations-manager). [jfsiii](http://github.com/jfsiii) will keep the branch up-to-date with `master` by doing a `git merge master` periodically (a few times a week?).
```

Comment:
Can you mention that the merges are done locally and then pushed because we can't work with a PR (merge permissions)?

Reply:
Added in f3c0664. Let me know if that looks ok.